src/pages/patientView/timeline/legacy.js

```diff
@@ -233,7 +233,10 @@ export function buildTimeline(
                     display: 'circle',
                     color: '#000',
                     tooltip_tables: [
-                        [['START_DATE', days], ['STATUS', 'DECEASED']],
```

Comment:
[Break arrays of arrays/objects if each element has more than one element/property](https://prettier.io/blog/2019/11/09/1.19.0.html#break-arrays-of-arraysobjects-if-each-element-has-more-than-one-elementproperty-6694-by-sosukesuzuki)